### PR TITLE
docs(guild): fix grammatical errors in Guild channel properties

### PR DESCRIPTION
**Pull request by toolifelesstocode**
## Summary

This pull request fixes the grammatical error in all the `Guild.*_channels` properties, as they currently say "... that belongs to..." instead of "... that belong to...".

## Checklist

- [ ] If code changes were made, then they have been tested
    - [ ] I have updated the documentation to reflect the changes
    - [ ] I have formatted the code properly by running `task lint`
    - [ ] I have type-checked the code by running `task pyright`
- [ ] This PR fixes an issue
- [ ] This PR adds something new (e.g. new method or parameters)
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [x] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

File: changelog/645.misc.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+Fix the grammatical error in ``Guild`` channel prepoerties.
```
Comment (shiftinv) [inline changelog/645.misc.rst:1]:
```suggestion
Fix the grammatical error in :class:``Guild`` channel properties.
```
This is also a docs change, `645.doc.rst` would be more suitable

Comment (toolifelesstocode) [reply changelog/645.misc.rst]: https://github.com/DisnakeDev/disnake/pull/645/commits/fe8096b8a341094e0b4793e92f468efa07dfa04e

File: changelog/645.doc.rst [hunk 2]
```diff
@@ -0,0 +1 @@
+Fix the grammatical error in :class:``Guild`` channel properties.
```
Comment (shiftinv) [inline changelog/645.doc.rst:1]:
```suggestion
Fix grammatical errors in :class:`Guild` channel properties.
```
missed this earlier, my bad

Comment (toolifelesstocode) [reply changelog/645.doc.rst]: Fixed in https://github.com/DisnakeDev/disnake/pull/645/commits/74577147f63c5d20cfd6fd8f7c5944727b0f039f.

**Review by shiftinv — APPROVED** [review]
lgtm, thanks!

File: changelog/645.doc.rst [hunk 3]
```diff
@@ -0,0 +1 @@
+Fix the grammatical error in :class:`Guild` channel properties.
```
Comment (onerandomusername) [inline changelog/645.doc.rst:1]: shouldn't this be plural?
```suggestion
Fix the grammatical errors in :class:`Guild` channel properties.
```

**Review by onerandomusername — APPROVED** [review]

File: changelog/645.doc.rst (hunk 3 above)
Comment (toolifelesstocode) [reply changelog/645.doc.rst]: https://github.com/DisnakeDev/disnake/pull/645/commits/080cfbd0a2ab0dedcda6f1b1a69ddc17d90828e8

**Pull request merged**